docs/polkadot/networking/3-avail-valid.md

```diff
@@ -35,9 +33,9 @@ Every block is erasure coded across N pieces with a threshold of ceil(N/3).
 
 At the start of the period, for every parachain, its N/C parachain validators each have all of the N pieces. In this role we call them the "preliminary checkers". The high-level purpose of A&V networking is to:
 
-1. Distribute the pieces of all C blocks to all other validators
+R1: Distribute the pieces of all C blocks to all other validators. A corollary of this is that every validator *must receive* at minimum (C-1) pieces, 1 from every other parachain.
```

Comment:
nit: This is predicated on each parachain including a block, which is not guaranteed.